nixos/modules/programs/rootston.nix

```diff
@@ -21,14 +22,28 @@ in {
       Wayland compositor you should e.g. use Sway instead). You can manually
       start the compositor by running "rootston" from a terminal'';
 
+    extraSessionCommands = mkOption {
+      type = types.lines;
+      default = "";
+      example = ''
+        # Define a keymap (US QWERTY is the default)
+        export XKB_DEFAULT_LAYOUT=de,us
+        export XKB_DEFAULT_VARIANT=nodeadkeys
+        export XKB_DEFAULT_OPTIONS=grp:alt_shift_toggle,caps:escape
```

Comment:
The preferred way to do this is in the config file.

Reply:
@emersion These options are (or were) not part of config. See e.g.
https://wiki.archlinux.org/index.php/Sway#Keymap
https://github.com/swaywm/sway/wiki#keyboard-layout

Reply:
rootston != sway

Reply:
@emersion Then feel free to fix it :-)

Reply:
Hi :) I think there was a bit of a misunderstanding.

@emersion I think you mean something like this (didn't check the syntax yet, just had a quick look at https://github.com/swaywm/wlroots/pull/341)?:
```ini
[keyboard]
layout = de,us
variant = nodeadkeys
options = grp:alt_shift_toggle,caps:escape
```
I'll have a closer look and make a PR later. Thank you for letting us know how we can improve this module (wasn't aware that this is possible) :smile:.

@gnidorah I assume (might be completely wrong) you where a bit "irritated" by the first comment (at least I initially was). @emersion is the main(?) `wlroots` developer and as I interpret his comment (now), is that he just wanted to help us.

Reply:
Yeah, sorry, I really didn't mean to be irritating. I just stopped by and saw you're still using the Sway way, which is supported for legacy reasons. It's no big deal in the end, feel free to do whatever you want. :)

Reply:
@emersion @primeos Ah, got it, thank you guys!